tensorflow/core/platform/s3/s3_file_system.cc

```diff
@@ -79,6 +79,14 @@ Aws::Client::ClientConfiguration& GetDefaultClientConfig() {
         cfg.verifySSL = true;
       }
     }
+    const char* connect_timeout = getenv("S3_CONNECT_TIMEOUT");
+    if (connect_timeout) {
+      cfg.connectTimeoutMs = atol(connect_timeout);
```

Comment:
Use safe_strto64 in tensorflow/core/lib/strings/numbers.h instead of atol.

Reply:
ACK